ucsb-cs156-s24/proj-happycows-s24-4pm-8
# rk- fixed BUG: Disallow ability to navigate to unjoined commons

## Overview

In this PR, I added a functionality to catch when users try to navigate to a commons that they are not a part of. 

## Screenshots
<img width="1512" alt="Screenshot 2024-05-28 at 10 25 34 AM" src="https://github.com/ucsb-cs156-s24/proj-happycows-s24-4pm-8/assets/92010151/b79c7361-9177-4dc3-8325-e13881ec4957">

## Future Possibilities 
- We could create a whole new page for users to be redirected to if they try to access a commons that they are not a part of. I just changed the header text on the page that they access when doing this. 
- We could also redirect users to a different error page when they try to access a commons that does not exist.

## Validation 

- login to happy cows on this deployment: https://proj-happycows-rk.dokku-08.cs.ucsb.edu
- Make a commons and do not join it 
- in the search bar, add /play/commonsID to the end of the url where commonsID is the id of the commons that you created but did not join 
- observe the message: "You have yet to join this commons!"

## Tests


## Linked Issues

Closes #10
